crypto/kyber/p_kyber512_asn1.c

```diff
@@ -103,11 +103,11 @@ static int kyber512_pub_cmp(const EVP_PKEY *a, const EVP_PKEY *b) {
 }
 
 static int kyber512_size(const EVP_PKEY *pkey) {
-  return KYBER512_PUBLIC_KEY_BYTES + KYBER512_PRIVATE_KEY_BYTES;
+  return KYBER512_PUBLIC_KEY_BYTES + KYBER512_SECRET_KEY_BYTES;
 }
 
 static int kyber512_bits(const EVP_PKEY *pkey) {
-  return 8 * (KYBER512_PUBLIC_KEY_BYTES + KYBER512_PRIVATE_KEY_BYTES);
+  return 8 * (KYBER512_PUBLIC_KEY_BYTES + KYBER512_SECRET_KEY_BYTES);
```

Comment:
huge nit, but still:
```suggestion
  return 8 * kyber512_size();
```

Reply:
Also, the documentation for the public function that uses this call-back is not very precise on what the expected value is for KEMs:
```
// EVP_PKEY_size returns the maximum size, in bytes, of a signature signed by
// |pkey|. For an RSA key, this returns the number of bytes needed to represent
// the modulus. For an EC key, this returns the maximum size of a DER-encoded
// ECDSA signature.
OPENSSL_EXPORT int EVP_PKEY_size(const EVP_PKEY *pkey);
```
Hopefully, the returned value is not going to change. But we'll see whether the final standard is going to introduce some encoding...

Reply:
added a sentence for KEMs